src/Toolkit/Toolkit.WinUI/UI/Controls/FloorFilter/FloorFilter.Resources.xaml

```diff
@@ -0,0 +1,8 @@
+﻿<ResourceDictionary
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:controls="using:Esri.ArcGISRuntime.Toolkit.UI.Controls"
+    xmlns:internal="using:Esri.ArcGISRuntime.Toolkit.Internal"
+    xmlns:toolkit="using:Esri.ArcGISRuntime.Toolkit.UI.Controls">
+    <FontFamily x:Key="CalciteIconFont">ms-appx:///Esri.ArcGISRuntime.Toolkit.WinUI/Assets/calcite-ui-icons-24.ttf#calcite-ui-icons-24</FontFamily>
```

Comment:
Just to nitpick: Why not just put this single line in `FloorFilter.Theme.xaml` instead of adding an entire file? (like you had it?) Also all the namespace declarations are probably not necessary.
The original idea of moving common resources to a common file really only applies if we could put it in a common resource file, but you said that didn't actually work, and if resources aren't really common after all (as in resources used by multiple controls), there's not much point in splitting things out into separate files. I was under the impression that the font file was used in multiple controls, and therefore should be in a single common place before any of the other theme files gets imported. Is floorfilter the only one using it?
Similar comment for the measure toolbar.

Might be simpler just to fix the URIs and not start moving a bunch of files around at this point.